tools/mason/MasonUpdate.chpl

```diff
@@ -42,7 +42,7 @@ private var failedChapelVersion : [1..0] string;
 
 /* Finds a Mason.toml file and updates the Mason.lock
    generating one if it doesnt exist */
-proc UpdateLock(args: [] string, tf="Mason.toml", lf="Mason.lock") {
+proc UpdateLock(args: [] string, tf="Mason.toml", lf="Mason.lock", quiet=false) {
```

Comment:
I'm not a fan of adding special-case arguments to the mason interface for testing purposes.

Long term, it'd probably be better to just support a `--quiet` flag, such that it's included in both the user and internal interface. In that case, `quiet` would be contained in `args`, and we could use it for most of testing.

Until then, I'd continue to use the `sed` commands in our prediff from before.

Reply:
Alternatively we could just unify the message for initializing and updating and get rid of the prediff.

 e.g. always say "Updating" (or something) even if the repo is technically being initialized

Reply:
@ronawho had a good idea: Change the "Initializing mason-registry for mason-registry" to just say "Updating mason-registry for mason-registry" too, so that we can just always expect that string, and drop all the prediffs.